test/remote/gateways/remote_checkout_v2_test.rb

```diff
@@ -1156,6 +1164,26 @@ def test_failed_verify
     assert_match %r{request_invalid: card_number_invalid}, response.message
   end
 
+  def test_successful_verify_with_ammount_allocations
+    response = @gateway.verify(@credit_card, @options_for_verify)
+    assert_success response
+    assert_match %r{Succeeded}, response.message
+  end
+
+  def test_successful_verify_via_oauth_with_ammount_allocations
+    response = @gateway_oauth.verify(@credit_card, @options_for_verify)
+    assert_success response
+    assert_match %r{Succeeded}, response.message
+    assert_not_nil response.responses.first.params['access_token']
+    assert_not_nil response.responses.first.params['expires']
+  end
+
+  def test_failed_verify_with_ammount_allocations
+    response = @gateway.verify(@declined_card, @options_for_verify)
+    assert_failure response
+    assert_match %r{request_invalid: card_number_invalid}, response.message
+  end
+
```

Comment:
Hey, I see that you added remote tests for verify and unit tests for authorize. Since verify is a $0 amount, and not completing a transfer of funds I think it would be prudent to add some remote tests with transaction types that actually move money.

Reply:
Also, nit pick but it looks like you've got an extra m in amount in your test definitions